tjreports/administrator/views/reports/tmpl/default.php

```diff
@@ -174,6 +174,8 @@
 					<input type="hidden" id="task" name="task" value="" />
 					<input type="hidden" name="boxchecked" value="0" />
 					<input type="hidden" name="totalRows" id="totalRows" value="<?php echo $this->items['total_rows']; ?>" />
+					<input type="hidden" name="extension" value="<?php echo $input->get('client','','word'); ?>">
+
```

Comment:
TJReports : TJReports is list view of all  types of reports (plugins). E.g : attemptreport,coursereport etc.

Reports : Reports is list of reports of particular report type (plugin) E.g only attempt report or any other.